src/components/visualisation/Heatmap.tsx

```diff
@@ -0,0 +1,212 @@
+import { VStack, HStack, Spacer, Box, Text, StackProps } from "@chakra-ui/react";
+import { ReactNode, useCallback, useMemo } from "react";
+import { prependApiUrl } from "utils/api/client";
+import "styles/atlas.css";
+import { RequiredAndNonNullable } from "schema/interfaces";
+
+const HEATMAP_PALETTE = ["#fde725", "#5ec962", "#21918c", "#3b528b", "#440154"];
```

Comment:
Can we only have `binCount` <= 5?

Reply:
Currently, yes, there isn't a place where more than 5 bins are used